### DISCUSSION TITLE
Status Bar Plugin - Display Future HP Value on Healing Item Hover

### DISCUSSION THREAD

**Discussion by theswede402** [runelite/runelite#16701] [Ideas]
I love the status bar plugin. Suggestion would be to have the value change - or an additional value displayed - when hovering over healing items. Currently the bar updates to show where your health will be but the value doesn't change. It would be nice to see the future value so I can quickly assess if I need to heal more to get above my opponent's max hit (and just to know where my health will be after healing).

**Comment by Nightfirecat** [comment]
I think the "Show restores" config option on the plugin does what you're asking for

**Reply by theswede402** [reply]
As far as I can tell, "show restores" only shows the bar filling up, but not the actual future HP value. See screenshot where my current HP is 40/50 and the bar shows the incremental heal but the number still shows 40.

![Status Bar](https://github.com/runelite/runelite/assets/127467487/9ad0ffb3-8ba9-4f1c-b765-8dc98f931bcb)

**Reply by Nightfirecat** [reply]
I think most users would not want this. You can already see in your screenshot that the item stats tooltip displays the resulting HP from eating the Salmon you're hovering, which I personally feel is sufficient already.
